### PR TITLE
Getting it to work

### PR DESCRIPTION
**Pull request by taylordowns2000**

### CODE REVIEW

File: ansible/roles/couchdb/tasks/main.yml [hunk 1]
```diff
@@ -13,6 +13,15 @@
     - libcurl4-gnutls-dev
     - libtool
 
+- name: Install couchdb pip packages
```
Comment (snopoke) [inline ansible/roles/couchdb/tasks/main.yml:16]: You might want to consider using couchdb2

File: ansible/roles/couchdb2/tasks/main.yml [hunk 2]
```diff
@@ -96,6 +97,17 @@
...
+# TASK [couchdb2 : monit] ********************************************************
+# fatal: [165.227.172.214]: FAILED! => {"changed": false, "failed": true, "msg": "couchdb2 process not presently configured with monit", "name": "couchdb2", "state": "monitored"}
+#
+# RUNNING HANDLER [monit : reload monit] *****************************************
+# 	to retry, use: --limit @/vagrant/ansible/deploy_stack.retry
+#
+# PLAY RECAP *********************************************************************
+# 165.227.172.214            : ok=36   changed=20   unreachable=0    failed=1
+# I get it to run after ssh-ing in and running `monit` as root, awakening the monit daemon.
+# I love awakening daemons...
```
Comment (snopoke) [inline ansible/roles/couchdb2/tasks/main.yml:110]: We've recently made some improvements to this role:
* https://github.com/dimagi/commcarehq-ansible/pull/966
* https://github.com/dimagi/commcarehq-ansible/pull/960

File: ansible/roles/couchdb/tasks/main.yml (hunk 1 above)
Comment (taylordowns2000) [reply ansible/roles/couchdb/tasks/main.yml:16]: Hey @snopoke , even on your current ansible master branch, it looks like you guys use both couchdb and couchdb2. Am I reading this right? Sorry for being thick—just not sure I understand what you mean here.

From line 42 of `deploy_db.yml` on master at commcarehq-ansible:
```yml
- name: Couchdb
  hosts: couchdb
  become: true
  roles:
    - {role: couchdb, tags: 'couchdb'}

- name: Couchdb log rolling configurations
  hosts: couchdb
  become: true
  roles:
    - role: ansible-logrotate
      tags: couchdb
      logrotate_scripts:
        - name: "{{ deploy_env }}_couchdb"
          path: "{{ couch_log_dir }}/*.log"
          options:
            - daily
            - size 100M
            - rotate 10
            - missingok
            - compress
            - delaycompress
            - copytruncate
            - nocreate
            - notifempty

- name: Couchdb 2.0
  hosts: couchdb2
  become: true
  roles:
    - {role: couchdb2, tags: 'couchdb2'}

- name: Couchdb2 log rolling configurations
  hosts: couchdb2
  become: true
  roles:
    - role: ansible-logrotate
      tags: couchdb2
      logrotate_scripts:
        - name: "{{ deploy_env }}_couchdb2"
          path: "{{ couchdb_dir }}/var/log/*.stderr"
          options:
            - daily
            - size 100M
            - rotate 10
            - missingok
            - compress
            - delaycompress
            - copytruncate
            - nocreate
            - notifempty
```

**Review by snopoke — COMMENTED** [review]
Yea, we have that because some of our environments still use couch1.6. Since you're on a new setup I'd only setup couch2.

You can do that by using `--skip-tags=couchdb2` or only running the `couchdb2` tag

File: ansible/roles/couchdb2/tasks/main.yml (hunk 2 above)
Comment (taylordowns2000) [reply ansible/roles/couchdb2/tasks/main.yml:110]: Thanks for the quick reply Just to be clear, do you mean `--skip-tags couchdb2` or `--skip-tags couchdb`?

Comment (snopoke) [reply ansible/roles/couchdb2/tasks/main.yml:110]: sorry , I mean `--skip-tags couchdb`

**Pull request merged**